src/tests/stats/test_distribution_uniform.f90

```diff
@@ -0,0 +1,855 @@
+program test_distribution_uniform
+    use stdlib_error, only : check
+    use stdlib_kinds
+    use stdlib_stats_distribution_PRNG, only : random_seed, dist_rand
+    use stdlib_stats_distribution_uniform,                                      &
+        uni_rvs => uniform_distribution_rvs,                                    &
+        uni_pdf => uniform_distribution_pdf,                                    &
+        uni_cdf => uniform_distribution_cdf
+
+    implicit none
+    logical ::  warn = .true.
+    real(sp), parameter :: sptol = 1000 * epsilon(1.0_sp)
+    real(dp), parameter :: dptol = 1000 * epsilon(1.0_dp)
+    real(qp), parameter :: qptol = 1000 * epsilon(1.0_qp)
+    integer :: put, get
+
+    put = 135792468
+
+    call test_shuffle
+
+    call test_uni_rvs_0
+
+    call test_uni_rvs_iint8
+    call test_uni_rvs_iint16
+    call test_uni_rvs_iint32
+    call test_uni_rvs_iint64
+    call test_uni_rvs_rsp
+    call test_uni_rvs_rdp
+    call test_uni_rvs_rqp
+    call test_uni_rvs_csp
+    call test_uni_rvs_cdp
+    call test_uni_rvs_cqp
+
+    call test_uni_pdf_iint8
+    call test_uni_pdf_iint16
+    call test_uni_pdf_iint32
+    call test_uni_pdf_iint64
+    call test_uni_pdf_rsp
+    call test_uni_pdf_rdp
+    call test_uni_pdf_rqp
+    call test_uni_pdf_csp
+    call test_uni_pdf_cdp
+    call test_uni_pdf_cqp
+
+    call test_uni_cdf_iint8
+    call test_uni_cdf_iint16
+    call test_uni_cdf_iint32
+    call test_uni_cdf_iint64
+    call test_uni_cdf_rsp
+    call test_uni_cdf_rdp
+    call test_uni_cdf_rqp
+    call test_uni_cdf_csp
+    call test_uni_cdf_cdp
+    call test_uni_cdf_cqp
```

Comment:
It seems that `fypp` could be used to generate all these subroutines.

Reply:
Agreed. But all test programs use .f90. If .fypp is used, the CMakeLists.txt has to be changed.

Reply:
Just realize the updated make file. Will change the test file.